packages/types/lib/appium-config.ts

```diff
@@ -6,9 +6,13 @@
  */
 
 /**
- * IP address to listen on
+ * IPv4/IPv6 address or a hostname to listen on
  */
-export type AddressConfig = string;
+export type AddressConfig = AddressConfig1 & AddressConfig2;
```

Comment:
I assume this should not be happening and it must be a bug in the types generator itself. Although I have no idea how I could influence that.

Reply:
maybe @boneskull could propose a solution?

Reply:
Since we're using `oneOf` here, you're going to have _n_ different types for each possible type.

What looks like a problem is the definition below of `AddressConfig1`:

```ts
export type AddressConfig1 = {
  [k: string]: unknown;
};
```

This is ostensibly coming from `format: 'ipv6'` , since we know that `hostname` translates to just `string`.  `ipv6` should _also_ be `string`.

Try changing `oneOf` to `anyOf`, rebuild, and check result.  The README for `json-schema-to-typescript` claims `oneOf` is "inexpressible".

Reply:
Unfortunately changing to anyOf did not help.
Now I get this error: 

```
Type 'string | undefined' is not assignable to type '(AddressConfig1 & string) | undefined'.
  Type 'string' is not assignable to type 'AddressConfig1 & string'.
    Type 'string' is not assignable to type 'AddressConfig1'.
```

Is it a dead end?